MetaData/data/MjjRegression/TMVA_mjj_reg_model_met_corr_All.xml

```diff
@@ -0,0 +1 @@
inaryTree><BinaryTree boostWeight="1.0e+00" itree="86" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.8298208e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="87" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.73887508e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="88" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.65101051e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="89" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.56756562e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="90" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.4900811e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="91" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.41701685e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="92" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.34549271e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="93" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.27838905e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="94" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.21436506e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="95" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.15476166e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="96" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.09669816e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="97" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="1.04305502e-06" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="98" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="9.90951662e-07" rms="0.0e+00" /></BinaryTree><BinaryTree boostWeight="1.0e+00" itree="99" type="DecisionTree"><Node Cut="0.0e+00" IVar="-1" NCoef="0" cType="1" depth="0" nType="-99" pos="s" purity="0.0e+00" res="9.41729184e-07" rms="0.0e+00" /></BinaryTree></Weights></MethodSetup>
```

Comment:
This file should not be here, it should go into the Taggers/data directory on EOS

Reply:
Done